HACKING.md

```diff
@@ -187,31 +187,29 @@ NOTE: For the prod pipeline these secrets can be found in BitWarden
 
 ### [OPTIONAL] Creating Azure credentials configs
 
-If you want to run kola tests against Azure images you need to
-create a secret with an `azureProfile.json` and a file called an
-azure auth file. See the
+If you want to do image uploads or run kola tests against Azure
+images you need to create a file called `azureCreds.json`. See the
 [kola docs](https://github.com/coreos/coreos-assembler/blob/main/docs/mantle/credentials.md#azure)
 for more information on those files.
 
-Once you have the azureAuth.json and azureProfile.json for connecting to Azure,
-create the secrets in OpenShift:
+Once you have the azureCreds.json for connecting to Azure, create the secrets in OpenShift:
 
 ```
-oc create secret generic azure-kola-tests-config-profile \
-    --from-literal=filename=azureProfile.json \
-    --from-file=data=/path/to/azureProfile.json
-oc label secret/azure-kola-tests-config-profile \
+oc create secret generic azure-image-upload-config \
+    --from-literal=filename=azure_config_file \
```

Comment:
A doubt after reading the comments above this line, does the azure_config_file refer to azureCreds.json here?

Reply:
`azure_config_file` is the value going into the `filename` key for this secret. when the secret gets used inside the pipeline it will expand to a file named `azure_config_file`, but we don't really care what the file is named. This is just a value that's consistent with what we are using for the other credentials for say GCP/AWS.